cmd/tracee-rules/input.go

```diff
@@ -145,7 +145,6 @@ func parseTraceeInputFile(option *traceeInputOptions, fileOpt string) error {
 	if err != nil {
 		return fmt.Errorf("invalid file: %s", fileOpt)
 	}
-	defer f.Close()
```

Comment:
I see. When this will be closed then?

Reply:
That's the entrypoint's job.

Reply:
also, why the e2e didn't fail once `defer f.Close()` was added?

Reply:
For some reason the workflows on the PR didn't run, not sure why.

Reply:
@mtcherni95 I saw it was a simple "fix" and the merge was green (I assumed tests had passed, which made me feel safe, but, for some reason we did not have tests in that PR, not sure why).